### PR TITLE
Added support for configurable aws profile 

### PR DESCRIPTION
**Pull request by wongJonathan**
Can now use `--aws-profile` or add `profile` to the serverless file under `provider`.

### CODE REVIEW

**Comment by lucasklaassen** [comment]
You guys are speedy!

File: index.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ const _ = require('underscore');
 class VPCPlugin {
   constructor(serverless) {
     this.serverless = serverless;
-    AWS.config.update({
-      region: serverless.service.provider.region,
-    });
+    const awsProvider = this.serverless.providers.aws.getCredentials();
```
Comment (aoskotsky-amplify) [inline index.js]: `awsCreds` might be a better name

Comment (wongJonathan) [inline index.js]: Yeah sure I'll change it.

**Comment by lucasklaassen** [comment]
I'll test this out on my project, thanks guys!

**Comment by wongJonathan** [comment]
No problem let us know how it goes!

**Pull request merged**